src/System.Net.Http/src/System/Net/Http/SocketsHttpHandler/HPack/StatusCodes.cs

```diff
@@ -0,0 +1,223 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0.
+// See THIRD-PARTY-NOTICES.TXT in the project root for license information.
+
+using System.Globalization;
+using System.Text;
+
+namespace System.Net.Http.HPack
+{
+    internal static class StatusCodes
+    {
+        private static readonly byte[] _bytesStatus100 = CreateStatusBytes((int)HttpStatusCode.Continue);
```

Comment:
It looks like all of these are eventually only consumed as read-only spans.  We should consider changing these to, e.g.
```C#
private static ReadOnlySpan<byte> BytesStatus200 => new byte[] { (byte)'2', (byte)'0', (byte)'0' };
```
and making the appropriate return-type change to ToStatusBytes and then the eventual consuming code in EncodeStatusCode.  That will end up being more efficient in both the binary and at run time.